Main.py

```diff
@@ -0,0 +1,117 @@
+window.blit(imagelistRan,(0, 0)) #Blits image to created window
+
+textlist = ["CJ?","Are you OK?","Excuse me Young Man!","WHEN YOU MEME"]
+textlist1 = ["Ohhhhhhhhhh","Big Smoke, It's me!","Boiiiiiiiiiiiiiiiiiiiiiiiii","It was me, Dio!","JAGUARS","BOTTOM TEXT"]#Picks random text snippet from list
+textlistRan = (random.choice(textlist))
+textlistRan1 =(random.choice(textlist1))
+
+def greyscale():
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+
+            grey = (Red + Green + Blue)/3
+
+            pxarray[x, y] = (grey, grey, grey)
+
+def redblack():
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+
+            spook = (Red + Green + Blue)/4
+            if Red > 25 and Green > 25 and Blue > 25:
+                pxarray[x, y] = (255, spook, spook)
+
+def invert(window):
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+            pxarray[x, y] = (255 - Red, 255 - Green, 255 - Blue)
+
+def nightvision(window):
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+            pxarray[x, y] = (255 - Red, Green, 255 - Blue)
+
+def ganggreen(window):
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+            pxarray[x, y] = (Red,255 - Green, Blue)
+
+def bluebell(window):
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+            pxarray[x, y] = (Red, Green, 255 - Blue)
+
+
+
+while True:
+
+    pxarray = pygame.PixelArray(window)  # Creates a pixel array on the existing image
+
+    for event in pygame.event.get():
+        if event.type == QUIT:
+            pygame.quit()
+            sys.exit()
+        if event.type == KEYDOWN and event.key == K_g:
+            greyscale()
+        if event.type == KEYDOWN and event.key == K_h:
+            redblack()
+        if event.type == KEYDOWN and event.key == K_j:
+            invert(window)
+        if event.type == KEYDOWN and event.key == K_k:
+            nightvision(window)
+        if event.type == KEYDOWN and event.key == K_f:
+            ganggreen(window)
+        if event.type == KEYDOWN and event.key == K_d:
+            bluebell(window)
+        
```

Comment:
You can create a short hand for event.type which would have made your if statements much shorter